### ISSUE TITLE
JS TestDriver causes Exception in IntelliJ 10.0.2

### ISSUE THREAD

**Issue by GoogleCodeExporter** [manojjsm/js-test-driver#201]
```
JSTestDriver Version 1.2

Starting and running the JSTestDriver Server is possible.
The Browser connects and you can start the tests to run.

Running JS TestDriver with IntelliJ 10.0.1 is working as expected.

When you start the test I receive the following error:

com.intellij.execution.testframework.sm.runner.SMTRunnerConsoleProperties.<init>
(Lcom/intellij/execution/configurations/RuntimeConfiguration;)V: 
com.intellij.execution.testframework.sm.runner.SMTRunnerConsoleProperties.<init>
(Lcom/intellij/execution/configurations/RuntimeConfiguration;)V
java.lang.NoSuchMethodError: 
com.intellij.execution.testframework.sm.runner.SMTRunnerConsoleProperties.<init>
(Lcom/intellij/execution/configurations/RuntimeConfiguration;)V
    at com.google.jstestdriver.idea.TestRunnerState.execute(Unknown Source)
    at com.intellij.execution.impl.DefaultJavaProgramRunner.doExecute(DefaultJavaProgramRunner.java:88)
    at com.intellij.execution.runners.GenericProgramRunner$1.execute(GenericProgramRunner.java:85)
    at com.intellij.execution.impl.ExecutionManagerImpl$2.run(ExecutionManagerImpl.java:157)
    at com.intellij.openapi.project.DumbServiceImpl.runWhenSmart(DumbServiceImpl.java:96)
    at com.intellij.openapi.project.DumbService$1.run(DumbService.java:76)
    at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.run(LaterInvocator.java:332)
    at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:209)
    at java.awt.EventQueue.dispatchEvent(EventQueue.java:597)
    at com.intellij.ide.IdeEventQueue.e(IdeEventQueue.java:652)
    at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:505)
    at com.intellij.ide.IdeEventQueue.b(IdeEventQueue.java:404)
    at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:368)
    at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:269)
    at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:184)
    at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:174)
    at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:169)
    at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:161)
    at java.awt.EventDispatchThread.run(EventDispatchThread.java:122)

```

Original issue reported on code.google.com by `marco.j...@gmail.com` on 23 Feb 2011 at 8:41

**Comment by GoogleCodeExporter** [comment]
```
Sry, plugin version i'm currently using is is 1.2.2b
```

Original comment by `marco.j...@gmail.com` on 23 Feb 2011 at 8:49

**Comment by GoogleCodeExporter** [comment]
```
This bug is fixed in JsTestDriver IDEA plugin v1.3 or later.
```

Original comment by `Sergey.S...@gmail.com` on 28 Jun 2011 at 2:27
- Changed state: **Fixed**
